target/plantbattles-1/WEB-INF/appengine-generated/datastore-indexes-auto.xml

```diff
@@ -0,0 +1,4 @@
+<!-- Indices written at Mon, 22 Jun 2020 22:39:54 UTC -->
+
+<datastore-indexes autoGenerate="true"/>
+
```

Comment:
Do we need to version control this file? if this is something that's auto generated, it might be good to add this to the .gitignore file. 

Same with the local_db.bin file, but I can't seem to leave a comment on that one lol.

Reply:
its all getting very ignored now